irods_browser_app/static/js/irods_signin.js

```diff
@@ -56,16 +70,28 @@ function irods_login() {
         },
         success: function(json) {
             if(json.irods_loggedin) {
-                var signInStr = "Signed in as " + json.user;
-                $("#sign-in-info").removeClass();
-                $("#sign-in-info").addClass("alert alert-info");
-                $("#sign-in-name").text(signInStr);
-                $("#irods_content_label").text(json.user);
-                $('#root_store').val(json.datastore);
-                $("#log-into-irods").hide();
-                $("#btn-signout-irods").show();
-                $("#btn-select-irods-file").toggleClass("hidden", false);
-                $("#irods-sel-file").text("No file selected.");
+                if ($("#irods-group").length) {
+                    // Logic for resource landing page
```

Comment:
the logic here and in create resource page are duplicated at least three times in this file... It's probably time to put them into a method and call it.

Reply:
I agree. But this could be an improvement we should handle in a subsequent ticket.

Reply:
I'm not sure it makes sense to kick this can down the road when it has all been touched in this branch.  I think the time to address it is now unless you have a ticket that has this work in it's scope.

Reply:
I vote to get this code out the door.

Reply:
@sblack-usu This will be implemented in a separate ticket as it requires more than consolidating the logic in .js files. Both uses differ in their template and so I think a consolidation of this functionality is the best way to go. Created a further ticket to address it: https://github.com/hydroshare/hydroshare/issues/3095